src/components/media-views.js

```diff
@@ -480,9 +481,9 @@ AFRAME.registerComponent("media-video", {
       this.mesh.material.needsUpdate = true;
     }
 
-    let texture;
+    let texture, audioSource;
     try {
-      texture = await this.createVideoTexture();
+      ({ texture, audioSource } = await this.createVideoTextureAndAudioSource());
```

Comment:
Was at first a bit confused by `audioSource` since its not an audio source but infact a video or audio element... Might be cleaner to have the audio source get created in that function, but this is super minor.